services/movie_session.py

```diff
@@ -2,6 +2,16 @@
 
 from db.models import MovieSession
 
+from db.models import Ticket
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
The type hint 'list[dict]' is used here, which is correct for Python 3.9 and above. Ensure that your environment supports this syntax. If not, consider using 'List[Dict]' from the 'typing' module.